### PR TITLE
Transfer analytics

### PR DESCRIPTION
**Pull request by dzucconi**
Closes https://github.com/aredotna/ervell/issues/1327

I got a lil in the weeds here looking trying to handle the routing. I need to tie the mutations to the pages directly then use the router on the server as well. But I'm leaving this alone for now.

Going to revisit this typography stuff with styled-system in a bit; same idea really though.

### CODE REVIEW

File: apps/actions/client/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const { data: { APOLLO, STATUS_CODE } } = sharify;
+
+module.exports = () => {
+  const mountPoint = document.getElementById('apolloMount');
+
+  const Component = {
+    UNAUTHORIZED: TransferChannelAccessDeniedPage,
+    NOT_FOUND: TransferChannelNotFoundPage,
+    OK: Routes,
+  }[STATUS_CODE];
```
Comment (dzucconi) [inline apps/actions/client/index.js]: This is just a quick hacky way of dealing with the errors for now.

File: apps/actions/templates/index.jade [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  != apollo.styles
+
+block body
+  #apolloMount
+    != apollo.html
+
+block scripts
+  script( src= asset('/assets/actions.js') )
+  script.
+    window.__APOLLO_STATE__ = !{JSON.stringify(apollo.state).replace(/</g, '\\u003c')};
```
Comment (dzucconi) [inline apps/actions/templates/index.jade]: I need to write a JSX layout.

Comment (dzucconi) [reply apps/actions/templates/index.jade]: Though I suppose we'd need a new search/user bar to do that.

Comment (broskoski) [reply apps/actions/templates/index.jade]: Yeah true.

**Comment by broskoski** [comment]
Looks great to me 💯

**Pull request merged**